rails/locale/az.yml

```diff
@@ -109,29 +109,29 @@ az:
       accepted: qəbul olunmalıdır
       blank: boş ola bilməz
       present: boş olmalıdır
-      confirmation: təsdiqə uygun deyil
+      confirmation: təsdiqə uyğun deyil
       empty: boş ola bilməz
       equal_to: "%{count}-ə bərabər olmalıdır"
       even: cüt olmalıdır
       exclusion: qorunur
       greater_than: "%{count}-dən böyük olmalıdır"
-      greater_than_or_equal_to: böyük və ya %{count}-ə bərabər olmalıdır
-      inclusion: siyahiyə daxil deyil
+      greater_than_or_equal_to: "%{count}-dən az olmamalıdır"
```

Comment:
meaning of the message was changed by dropping "equal to"

Reply:
_greater than or equal to n_ means n is our minimum acceptable value. expressing such way (n-dən az olmamalıdır) in azerbaijani, i guess is more clear.

Reply:
That way it is not clear if n is included or excluded. The message should be explicit. See Turkish for example.